opm/input/eclipse/Schedule/HandlerContext.hpp

```diff
@@ -166,7 +166,7 @@ class HandlerContext
     const DeckKeyword& keyword;
     const std::size_t currentStep;
     const Action::Result::MatchingEntities& matches;
-    const bool actionx_mode;
+    const bool wellpi_action_mode;
```

Comment:
Tiny nit, but this renaming (`actionx_mode` :arrow_right: `wellpi_action_mode`) is a little bit misleading.  It absolutely used to be the case that `WELPI` handling was the only thing that cared about whether or not it was being applied in an action context.  However, following PR #4343 there are more things that care about whether or not they're running in an action context.  Putting `welpi` into the name is a little confusing for those.